tests/unit/test_jobs.py

```diff
@@ -2871,6 +2871,47 @@ def __init__(self):
             ],
             {},
         ),
+        # TODO
```

Comment:
I think these tests should not be extended as the check happens in the pre_check